validator/testdata/transformed_feature_tests/i_amphtml_sizer_child.out

```diff
@@ -42,6 +42,10 @@ FAIL
 |     <amp-video-iframe id="myVideo" src="/amp-video-iframe-videojs.html" width="720" height="405" layout="responsive" autoplay="" class="i-amphtml-layout-responsive i-amphtml-layout-size-defined" i-amphtml-layout="responsive">
 |       <i-amphtml-sizer style="display:block;padding-top:56.2500%;"></i-amphtml-sizer>
 |       <amp-img placeholder="" src="/img/amp-video-iframe-sample-placeholder.jpg" layout="fill" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
+>>      ^~~~~~~~~
+transformed_feature_tests/i_amphtml_sizer_child.html:43:5 The mandatory attribute 'style' is missing in tag '$reference_point'. (see https://amp.dev/documentation/components/amp-video-iframe/)
+>>      ^~~~~~~~~
+transformed_feature_tests/i_amphtml_sizer_child.html:43:5 The tag 'amp-img', a child tag of 'AMP-VIDEO-IFRAME (transformed)', does not satisfy the reference point 'AMP-VIDEO-IFRAME > I-AMPHTML-SIZER [style]'. (see https://amp.dev/documentation/components/amp-video-iframe/)
```

Comment:
I'm unsure of this side effect since I did not write the transformed validation spec.

Reply:
@alanorozco here's a better implementation to handle the transformed version of amp-video-iframe. cl/361232007

Reply:
Perfect, thanks! spec and tests now match.